pkg/yttlibrary/assert.go

```diff
@@ -25,6 +26,20 @@ var (
 
 type assertModule struct{}
 
+func (b assertModule) Equals(thread *starlark.Thread, f *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	if args.Len() != 2 {
+		return starlark.None, fmt.Errorf("expected two arguments")
+	}
+
+	expected, _ := core.NewStarlarkValue(args.Index(0)).AsString()
+	actual, _ := core.NewStarlarkValue(args.Index(1)).AsString()
```

Comment:
Before this is merged, we probably want to check these errors and report them.

Reply:
I removed the conversion to String and it removes the need for a return value here. This might also further the "motivating" case for this feature that you commented on below.